src/mongocxx/collection.cpp

```diff
@@ -65,6 +65,23 @@ stdx::string_view collection::name() const noexcept {
     return _impl->name;
 }
 
+void collection::name(stdx::string_view new_name, bool overwrite_old) {
+  bson_error_t error;
+
+  _impl->name = (std::string)new_name;
```

Comment:
@amidvidy I initially did not use the cast, but it failed. Further investigation revealed that the string conversion operator in mnmlstc is marked explicit.

https://mnmlstc.github.io/core/string.html#basic_string_view:T:::castto-std::basic_string:T:-operator